spec/features/gym_search/index_spec.rb

```diff
@@ -23,4 +23,13 @@
       end
     end
   end
+
+  it 'has a link to each individual gym', :vcr do
```

Comment:
Hardcode makes me nervous, would like to come back to if time (set as a vars in a before block or something)

Reply:
I agree!